dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/note/DefaultTrackedEntityCommentService.java

```diff
@@ -45,15 +45,15 @@ public class DefaultTrackedEntityCommentService implements TrackedEntityCommentS
 
   @Override
   @Transactional
-  public long addTrackedEntityComment(TrackedEntityComment comment) {
+  public long addTrackedEntityComment(Note comment) {
```

Comment:
We should change all the `TrackedEntityComment` to `Note` and also the method variable names